easybuild/easyconfigs/a/AOMP/build_aomp-13.0-2.patch

```diff
@@ -0,0 +1,218 @@
+--- a/aomp/bin/aomp_common_vars
```

Comment:
Please add a short description on top of the patch file, and mention where the patch comes from, or who created it (and keep in mind that this implies changing the SHA256 checksum in the easyconfig file)

Reply:
Fixed